bundle/tests/scorecard/kuttl/test-iqe-jobs/01-pods.yaml

```diff
@@ -77,4 +77,6 @@ spec:
       marker: "smoke"
       dynaconfEnvName: "clowder_smoke"
       filter: "test_plugin_accessible"
-
+      requirements: "{\"some\",\"requirements\"}"
```

Comment:
Could this be expressed with single quotes to avoid the escaping, just like is used in the assertions?

Reply:
Yeah, I did that on purpose for variety to make sure in the assertion it came out as expected :)